src/fridge.py

```diff
@@ -1,42 +1,69 @@
+from datetime.datetime import now
 import torch
 
 
-def model_saver(model, path):
+def save_model(checkpoint_num, epoch_num, gen, disc, optimizer_gen,
+               optimizer_disc):
     '''
-    NOTE: WIP/TODO
-
-    Takes in a generator or descriminator model and saves the model to an
-    appropriate location for future generation/discrimination
+    Saves model checkpoint.
 
     Parameters
     ----------
-    model: a trained pytorch model
-    path: path to where to save the model
-
-    Returns
-    -------
-    True if it worked false otherwise, possibly something else to be decided
-    later
+    checkpoint_num : int
+        Checkpoint number.
+    epoch_num : int
+        Epoch number.
+    gen : torch.nn.Module
+        Generator object.
+    disc : torch.nn.Module
+        Discriminator object.
+    optimizer_gen : torch.optim.Optimizer
+        Generator optimizer object.
+    optimizer_disc : torch.optim.Optimizer
+        Discriminator optimizer object.
     '''
-    torch.save(model.state_dict(), location)
-    return true
+    now = now().strftime("%m.%d.%H.%M")
+    model = {
+        'epoch': epoch,
+        'gen': gen,
+        'disc': disc,
+        'optimizer_gen': optimizer_gen,
+        'optimizer_disc': optimizer_disc,
+    }
+    state = {
+        'epoch': epoch,
+        'gen_state_dict': gen.state_dict(),
+        'disc_state_dict': disc.state_dict(),
+        'optimizer_gen_state_dict': optimizer_gen.state_dict(),
+        'optimizer_disc_state_dict': optimizer_disc.state_dict(),
+    }
+    torch.save(model, 'checkpoints/models/model_{}_{}.pt'.format(
```

Comment:
I think we're using python3.6? If so please use f-strings. 

https://www.python.org/dev/peps/pep-0498/

Reply:
We're only py3.5 on kahan.